.github/workflows/ci.yml

```diff
@@ -37,19 +39,24 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Test
-        if: github.ref != 'refs/heads/master'
-        run: npm run test
-
-      - name: Test & Coverage
-        if: github.ref == 'refs/heads/master'
+      - name: Tests & Coverage
         run: |
           npm run test
-          export COVERALLS_REPO_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }}
           npm run coverage
         env:
           CI: true
 
+      - name: Sonar Code Analysis
+        if: "github.repository == 'pioardi/poolifier'"
+        run: |
+          pwd
+          cd lib/
+          ls
```

Comment:
Looks like debugging, is this still needed?

Reply:
Going to remove those